jekyll/_sass/_sidebar.scss

```diff
@@ -2,7 +2,7 @@ $sidebar-spacing-base: 5px;
 $sidebar-width: 260px;
```

Comment:
If this variable isn't used anywhere after the change to `$leftbar-width`, it should be removed.